opta/cli.py

```diff
@@ -61,6 +62,7 @@ def cli() -> None:
         # However, we should still clean them up before the next command, or
         # else it may interfere with it.
         one_time()
+        pre_check()
```

Comment:
@rabbitfang you can use the same method for your pre-req checks

Reply:
@rabbitfang I have kept the pre_check.py, but have removed the definition from there. Please add the definition as and when required.